test/tests/functional/pbs_reservations.py

```diff
@@ -2268,7 +2268,7 @@ def test_resv_reconfirm_holding_partial_nodes(self):
         self.server.manager(MGR_CMD_SET, SERVER, a)
         a = {'resources_available.ncpus': 2}
         self.mom.create_vnodes(a, num=3)
-        vn_list = ['vn[0]', 'vn[1]', 'vn[2]']
+        vn_list = ["%s[%d]" % (self.mom.shortname, i) for i in range(3)]
```

Comment:
May for future we should change signature of create_vnodes to return either vnode names or list of newly create vnode objects (aka MoM objects or introduce new Vnode object in PTL)

Reply:
I'll +1 returning a list of vnode names.  I think they'll be more useful than creating a vnode object, but that works too.